iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/TransportClientTests.java

```diff
@@ -956,7 +973,7 @@ private void tearDownTwin(TransportClient transportClient)
         catch (Exception e)
         {
             e.printStackTrace();
-            fail("Encountered exception during tearDownTwin: " + Tools.getStackTraceFromThrowable(e));
+            TestCase.fail("Encountered exception during tearDownTwin: " + Tools.getStackTraceFromThrowable(e));
```

Comment:
why do we want tests to fail if teardown fails? Anything that is relevant to the test case should be a part of the test method directly.

Reply:
Good point. I'll remove this